nomad/structs/structs.go

```diff
@@ -11914,6 +11914,7 @@ type ACLTokenDeleteRequest struct {
 type ACLTokenBootstrapRequest struct {
 	Token      *ACLToken // Not client specifiable
 	ResetIndex uint64    // Reset index is used to clear the bootstrap token
+	Secret     string
```

Comment:
Let's call this `BootstrapSecret`, it's redundant with the struct name, but the fact that we have a weird `Token` field in here makes it non-obvious that this field *is* client-settable.

Reply:
OK will do